apps/dotcom/client/src/tla/components/TlaEditor/editor-messages.ts

```diff
@@ -5,4 +5,5 @@ export const editorMessages = defineMessages({
 	untitledProject: { defaultMessage: 'Untitled file' },
 	downloadFile: { defaultMessage: 'Download file' },
 	copyToMyfiles: { defaultMessage: 'Copy to my files' },
+	anonymousUser: { defaultMessage: 'Guest User' },
```

Comment:
If I can help wordsmith here a bit, I think Guest User would be better serviced by just simply saying "Guest".

Reply:
![qttymueesbw71](https://github.com/user-attachments/assets/bc26fc30-fd74-4937-9dca-1a80566a28f2)

Reply:
tried this, it doesn't work quite as well. when you add 'user' in there it feels a tad crufty yes but in this situation that actually helps to convey that it's a default value provided by the app rather than a name chosen by the user, so it parses easier.